server/channels/app/product.go

```diff
@@ -71,6 +72,11 @@ func (s *Server) initializeProducts(
 }
 
 func (s *Server) shouldStart(product string) bool {
+	if s.skipProductsInit && product != "channels" {
```

Comment:
Can we remove the `MM_DISABLE_BOARDS` and `MM_DISABLE_PLAYBOOKS`below as well?

Reply:
I don't think they're in use anywhere, but I'd rather do it as a follow up PR in case we'd like to revert it independently for some reason

Reply:
Let's leave those in place for a cycle or two - just in case.